spec/registry.xml

```diff
@@ -243,11 +243,72 @@
       </option>
     </formatSignature>
 
+    <matchSignature>
+      <input validationRule="anyNumber"/>
+      <option name="select" values="plural ordinal exact" default="plural">
+        <description>
+          The number selection type.
+
+          With `exact` selection, only a variant with a key matching
+          the JSON representation of the input number (without an exponent) will be selected.
+          With `plural` and `ordinal` selection, if no such exactly matching key is present,
+          a variant with the matching CLDR plural category (`zero`/`one`/`two`/`few`/`many`/`other`)
+          for the corresponding selection type will be selected,
+          according to the rules of the current locale
+        </description>
+      </option>
+      <option name="minimumIntegerDigits" values="positiveInteger" default="1">
+        <description>
+          The minimum number of integer digits to use.
+          A value with a smaller number of integer digits than this number will be
+          left-padded with zeros (to the specified length) when formatted.
+        </description>
+      </option>
+      <option name="minimumFractionDigits" values="positiveInteger">
+        <description>
+          The minimum number of fraction digits to use.
+          The default for plain number and percent formatting is 0;
+          the default for currency formatting is the number of minor unit digits provided by
+          the ISO 4217 currency code list (2 if the list doesn't provide that information).
```

Comment:
This is more complicated than it looks. There are several currencies with variable presentation (official vs. customary). In most cases, the currency overrides the fraction digits when it is set and we don't have to be overly specific, so perhaps:

```suggestion
the default for currency formatting depends on the currency code selected.
```

Reply:
This is a copy of the text used for the corresponding formatting option. Could we take on as a separate task a review of these option descriptions?

Reply:
I saw that. I suppose we could review separately, but why not get it out of the way? But I could live with it.

Reply:
Let's follow up on this under #596.